### PR TITLE
Fix example code in Realtime Guide

### PR DESCRIPTION
**Pull request by kevincam3**
Just needed to use object destructuring and not JSON.stringify to get the data.

### CODE REVIEW

File: docs/guides/real-time/getting-started/websockets.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ connection.addEventListener('open', function () {
 });
 
 connection.addEventListener('message', function (message) {
-	const { data } = JSON.parse(message);
+	const { data } = message;
```
Comment (br41nslug) [inline docs/guides/real-time/getting-started/websockets.md:60]: The original example was indeed wrong, however `message.data` is still a string containing the JSON response.

The correct solution here is to parse `message.data`:
```suggestion
	const data = JSON.stringify(message.data);
```
https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/message_event#event_properties

**Review by br41nslug — DISMISSED** [review]
Thanks for pointing out the incorrect example in the docs 😄

File: docs/guides/real-time/getting-started/websockets.md (hunk 1 above)
Comment (paescuj) [reply docs/guides/real-time/getting-started/websockets.md:60]: Did you mean? 😄 
```suggestion
	const data = JSON.parse(message.data);
```

Comment (br41nslug) [reply docs/guides/real-time/getting-started/websockets.md:60]: Lol, guess i didnt have enough coffee yet 😂

Comment (paescuj) [reply docs/guides/real-time/getting-started/websockets.md:60]: 😀
monday

**Review by paescuj — APPROVED** [review]

**Review by br41nslug — APPROVED** [review]

**Pull request merged**